pkgs/misc/emulators/higan/default.nix

```
@@ -1,135 +1,158 @@
-{ lib, stdenv, fetchFromGitHub
-, pkg-config
-, libX11, libXv
-, udev
-, libGLU, libGL, SDL2
-, libao, openal, libpulseaudio
+{ lib
+, stdenv
+, fetchFromGitHub
+, SDL2
 , alsa-lib
-, gtk2, gtksourceview
+, gtk3
+, gtksourceview3
+, libGL
+, libGLU
+, libX11
+, libXv
+, libao
+, libpulseaudio
+, openal
+, pkg-config
 , runtimeShell
+, udev
 # Darwin dependencies
-, libicns, Carbon, Cocoa, OpenGL, OpenAL}:
+, libicns
+, Carbon
+, Cocoa
+, OpenAL
+, OpenGL
+}:
 
 let
   inherit (lib) optionals;
 in
 stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation rec {
```
lib is very short :)